src/api.py

```diff
@@ -0,0 +1,18 @@
+import requests 
+import hashlib
+def fetch(url):
+    con = requests.head(url)
```

Comment:
Why to do issue a head first?

Reply:
I am not able to get this please can you elaborate a little :sweat_smile: